MoviesMVVM/MoviesMVVM/Proxy/Proxy.swift

```diff
@@ -0,0 +1,22 @@
+// Proxy.swift
+// Copyright © RoadMap. All rights reserved.
+
+import Foundation
+
+/// Сервис для скачивания фото
+final class Proxy: Proxable {
```

Comment:
```suggestion
final class Proxy: Proxable {
```

тоже название не нравится. представь что тебе нужно изменения внести не в свой код, а в мой. какой бы ты класс искал для загрузки изображений? ImageService какой нибудь, врятли прокси. Прокси не относится конкретно к изображениям, например ты можешь кешировать данные которые у тебя с нетворксервиса приходят... не писать же прокси2

Reply:
так нам говорили за прокси, вот прокси он и есть